nodes-media/src/controllers/services/shareImagePuppeteer.ts

```
@@ -0,0 +1,942 @@
+        text: text as string,
+        answer: answer as string,
+        citations: citationsData,
+        refs: referenceCount,
+      });
+    } catch (imageError) {
+      if (imageError instanceof PuppeteerFallbackError) {
+        logger.info('Puppeteer failed for URL parameters, falling back to SVG approach');
+        return await useSvgFallback(req, res);
+      }
+      throw imageError;
+    }
```

Biome (2.1.2)

[error] This code is unreachable



(lint/correctness/noUnreachable)